Remotion/Web/Core/Res/Themes/NovaGray/HTML/WebButton.css

```diff
@@ -96,3 +96,8 @@ button.webButton[disabled] span.buttonBody img
   filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
           progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
 }
+
+.mozilla button.webButton span.buttonBody

```

Comment:
wouldn't the -2px be a bug that should be fixed globally isntead of just for Firefox?

Reply:
In Chrome (for example), the `left: -2px;` setting is needed to make sure WebButtons and normal buttons are aligned.

### Chrome with `left: -2px`
![image](https://user-images.githubusercontent.com/40357550/98242374-ad56ad80-1f6c-11eb-970c-26ba0548c806.png)

In Firefox however, the setting breaks the alignment now.

### Firefox with `left: -2px`
![image](https://user-images.githubusercontent.com/40357550/98242501-decf7900-1f6c-11eb-9e89-2e2c3fa15687.png)

Reply:
Interesting, as you can see in the above pictures, buttons in chrome do not span over the whitespace after them, in Firefox they do.

Reply:
headache. next week pairsession?